.pylintrc

```diff
@@ -10,3 +10,9 @@ disable=
     too-many-locals,
     # handled by black
     format
+
+[SIMILARITIES]
+min-similarity-lines=6
```

Comment:
why is this so high?

Reply:
The default is 4, I think. Reverted back to the default and added comments to avoid the false positives.